applications/StructuralMechanicsApplication/custom_response_functions/adjoint_elements/adjoint_finite_difference_base_element.cpp

```diff
@@ -292,6 +292,15 @@ void AdjointFiniteDifferencingBaseElement::CalculateSensitivityMatrix(const Vari
     // Get pseudo-load from utility
     ElementFiniteDifferenceUtility::CalculateRightHandSideDerivative(*pGetPrimalElement(), RHS, rDesignVariable, delta, rOutput, process_info);
 
+    if (rOutput.size1() == 0 || rOutput.size2() == 0)
+    {
+        const SizeType number_of_nodes = mpPrimalElement->GetGeometry().PointsNumber();
+        const SizeType dimension = rCurrentProcessInfo.GetValue(DOMAIN_SIZE);
+        const SizeType num_dofs_per_node = (mHasRotationDofs) ?  2 * dimension : dimension;
+        const SizeType local_size = number_of_nodes * num_dofs_per_node;
+        rOutput = ZeroMatrix(dimension * number_of_nodes, local_size);
```

Comment:
This does not fit to the description of the PR. There you indicated that the number of rows of a double variable has to be the same as the number of nodes.

Reply:
:+1: thanks for spotting